src/videouplinkbandwidthpolicy/DefaultSimulcastUplinkPolicy.ts

```diff
@@ -128,12 +130,24 @@ export default class DefaultSimulcastUplinkPolicy implements SimulcastUplinkPoli
       this.lastUplinkBandwidthKbps >= hysteresisIncrease ||
       this.lastUplinkBandwidthKbps <= hysteresisDecrease
     ) {
-      if (this.numParticipants >= 0 && this.numParticipants <= 2) {
-        // Simulcast disabled
+      if (this.shouldDisableSimulcast) {
+        // See comment above `shouldDisableSimulcast` for usage.
+        //
+        // The value of `newActiveStreams` is somewhat irrelevant since this single
+        // stream will adapt based on both sender and receiver network conditions.
+        //
+        // We use the middle stream here to work around a bug in Chromium where
+        // it seems when a transceiver is created when BWE is low (e.g. on a reconnection),
+        // it will never reset the encoder even when `setParameters` is called.  WebRTC bug
+        // #12788 seems to call a similar issue out as fixed for VP8, it's not clear if this
+        // is the same issue for H.264. Additionally we are not able to force a keyframe
+        // request from the backend since it will only be sending padding (which also
+        // don't have MID due to #10822). Since we don't scale when simulcast is disabled
+        // this doesn't have any end-user effect.
         this.newActiveStreams = ActiveStreams.kHi;
         newBitrates[0].maxBitrateKbps = 0;
-        newBitrates[1].maxBitrateKbps = 0;
-        newBitrates[2].maxBitrateKbps = 1200;
+        newBitrates[1].maxBitrateKbps = 1200;
```

Comment:
Curious. Can't it be the 1st (index 0) stream?

Reply:
With the recent fixes, we're switching back from our fork to the canonical AWS repo, but keeping our own uplink policy (and merging in your changes). With that in mind, I have a question to help me understand:

The comment above says: 
"  // The value of `newActiveStreams` is somewhat irrelevant since this single
   // stream will adapt based on both sender and receiver network conditions."
       
So this means it's OK to leave 'newActiveStreams' to kHi, while only populating the medium stream index[1]?

Reply:
I actually re-read that comment and also thought it was a bit confusing so i rewrote it in https://github.com/aws/amazon-chime-sdk-js/pull/1900 (also note I added another comment to the bottom of that block).

But to answer your question, yes, and it only effects the `encodingSimulcastLayersDidChange` callback to your application on the sender side (which currently is purely informational).